console.lua

```diff
@@ -187,6 +187,73 @@ commands = {{
         end
         return "Discards are now " .. G.GAME.current_round.discards_left
     end
+}, {
+    name = "joker",
+    source = "debugplus",
+    shortDesc = "Spawns a joker",
+    desc = [[Spawns Jokers. Usage:\njoker {args} [name]\n 
```

Comment:
Might be nice to have a subcommand to clear the jokers, for the case where you accidentally spawn 1000 jimbo's and your game is lagging really bad.